aea/cli/registry/add.py

```diff
@@ -28,6 +28,20 @@
 from aea.configurations.base import PublicId
 
 
+def _get_package_meta(obj_type: str, public_id: PublicId) -> dict:
```

Comment:
In my opinion this should be moved to `aea.cli.registry.utils` and made a public function (similar to `download_file`) so it can be reused.

Reply:
done